gcs/src/components/fla/graph.jsx

```diff
@@ -202,10 +205,15 @@ export default function Graph({ data, events, flightModes, graphConfig }) {
         if (flightMode.name === 'MODE') {
           flightModeColorsMap[flightMode.Mode] = colors[index]
         } else if (flightMode.name === 'HEARTBEAT') {
-          // TODO: Change if plane
-          flightModeColorsMap[
-            COPTER_MODES_FLIGHT_MODE_MAP[flightMode.custom_mode]
-          ] = colors[index]
+          if (flightMode === 1) {
```

Comment:
would be nice to have some comments here to quickly explain this block as its a bit bunched up rn